### PR TITLE
Feature/replace sqlite with postgresql

### PR DESCRIPTION
**Pull request by BeigeBadger**

### CODE REVIEW

File: azuresite/settings.py [hunk 1]
```diff
@@ -84,8 +84,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'taa_portal',
+        'USER' : 'pandaadmin@pg-panda-taa-sea-dev',
```
Comment (BeigeBadger) [inline azuresite/settings.py:89]: We should avoid putting secrets in source control

File: requirements.txt [hunk 2]
```diff
@@ -1,14 +1,8 @@
 asgiref==3.3.2
 Django==3.1.7
-# This is the most recent version that can be used with Python 3.6
-# As per https://docs.scipy.org/doc/scipy/reference/toolchain.html
-# the version history is here https://pypi.org/project/numpy/#history
 numpy==1.19.5
```
Comment (BeigeBadger) [inline requirements.txt]:
```suggestion
# This is the most recent version that can be used with Python 3.6
# As per https://docs.scipy.org/doc/scipy/reference/toolchain.html
# the version history is here https://pypi.org/project/numpy/#history
numpy==1.19.5
```

File: requirements.txt [hunk 3]
```diff
@@ -1,14 +1,8 @@
...
-# This is the most recent version that can be used with Python 3.6
-# As per https://pypi.org/project/pandas/1.1.5/ (see the Programming Language
-# section on the left) the version history is here
-# https://pypi.org/project/pandas/#history
 pandas==1.1.5
```
Comment (BeigeBadger) [inline requirements.txt:11]:
```suggestion
# This is the most recent version that can be used with Python 3.6
# As per https://pypi.org/project/pandas/1.1.5/ (see the Programming Language
# section on the left) the version history is here
# https://pypi.org/project/pandas/#history
pandas==1.1.5
```

**Review by BeigeBadger — COMMENTED** [review]
See my changes please

**Review by fahmie78 — APPROVED** [review]
I approved this changes

**Pull request merged**